examples/lock-app/nrfconnect/main/include/CHIPProjectConfig.h

```diff
@@ -31,4 +31,4 @@
 #define CHIP_DEVICE_CONFIG_USE_TEST_SETUP_PIN_CODE 20202021
 #define CHIP_DEVICE_CONFIG_USE_TEST_SETUP_DISCRIMINATOR 0xF00
 
-#define CHIP_DEVICE_CONFIG_SED_IDLE_INTERVAL 20000_ms32
+#define CHIP_DEVICE_CONFIG_SED_IDLE_INTERVAL 1000_ms32
```

Comment:
Locks that we've worked with tend to have longer sleep intervals in the order of 5-20s.